src/rgw/rgw_lc.cc

```diff
@@ -147,10 +147,9 @@ void *RGWLC::LCWorker::entry() {
 
     utime_t end = ceph_clock_now();
     int secs = schedule_next_start_time(start, end);
-    time_t next_time = end + secs;
-    char buf[30];
-    char *nt = ctime_r(&next_time, buf);
-    dout(5) << "schedule life cycle next start time: " << nt <<dendl;
+    end += secs;
```

Comment:
Hm overloading + with double in `utime_t` returning utime_t may not be the best option as we have code in many places that does a utime_t + double and uses the implicit `double` conversion. 

Another option is something in the lines of 

```c++
utime_t next; 
next.set_from_double(end+secs)
```

(we could make a  utime_t constructor to construct from double, not sure whether that is ok to introduce at this point)